test/test-data-unicode.json

```diff
@@ -898,5 +898,11 @@
     "name": "SyntaxError",
     "message": "Invalid escape sequence at position 1\n    \\u{110000}\n     ^",
     "input": "\\u{110000}"
+  },
+  "\\a": {
+    "type": "error",
+    "name": "SyntaxError",
+    "message": "atomEscape at position 1\n    \\a\n     ^",
+    "input": "\\a"
```

Comment:
Please add a test in `test-data.json` showing that `\\a` parses properly (without an exception) when the `u` flag is not set.